src/FsAutoComplete.Core/Commands.fs

```diff
@@ -924,57 +940,54 @@ type Commands<'analyzer> (serialize : Serializer, backgroundServiceEnabled) =
         return CoreResponse.Res res
     }
 
-    member x.GetUnusedDeclarations file = async {
-        let file = Path.GetFullPath file
-        let isScript = file.EndsWith ".fsx"
+    member x.CheckUnusedDeclarations file: Async<unit> =
```

Comment:
all of the notification-style members had a few stylistic changes:
* rename to `Check` instead of `Get` to mark that they have sideeffects
* return types changed to `Async<unit>` instead to doubly-signify that they have sideeffects